lib/repl.js

```diff
@@ -952,9 +953,4 @@ function isRecoverableError(e) {
   return e &&
       e.name === 'SyntaxError' &&
       /^(Unexpected end of input|Unexpected token)/.test(e.message);
-}
-
-function Recoverable(err) {
```

Comment:
I think there is at least one place left in the code that calls `new Recoverable()`.